dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/helper/InstrumenterActivator.java

```diff
@@ -19,14 +19,24 @@
 import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
 import io.opentelemetry.instrumentation.api.instrumenter.InstrumenterBuilder;
 
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+
 public class InstrumenterActivator implements BundleActivator {
+
+    private static final TraceComponent tc = Tr.register(InstrumenterActivator.class);
+
     //The J2S exception occurs due to a call during the initialization of Instrumenter and InstrumenterBuilder
     @Override
     public void start(BundleContext ctx) throws Exception {
         Class.forName(Instrumenter.class.getName());
         Class.forName(InstrumenterBuilder.class.getName());
         // Eagerly initialize context classes to avoid J2S exceptions
-        Context.root();
+        try {
+            Context.root();
+        } catch (Exception e) {
+            Tr.error(tc, Tr.formatMessage(tc, "CEMOT5101.telemetry.error", e));
+        } 
```

Comment:
I don't think we should add a try-catch here. This would occur at startup, and I don't think we can do anything sensible if this fails.